PocketExample/Pocket/Models/Wallet.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Wallet.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 4/24/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+
+struct Wallet {
+    var address = ""
+    var privateKey = ""
+    var network: Network?
+
+    func createWalletWith(passphrase: String, network: Network) -> Wallet {
+        // Create Wallet with passphrase
```

Comment:
If you're gong to document the PR, which you definitely should, you should follow Swift Markup syntax.

https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html

Please write the comment above the function, and add a description, parameter documentation, and return type documentation.

The link above is a primer. Here's an example from an OSS lib of mine: https://github.com/ArtSabintsev/Guitar/blob/master/Sources/GuitarPadding.swift#L33